### PR TITLE
Create categories

### PR DESCRIPTION
**Pull request by keccers**
*Add controller, view, models to allow categories to be shown 
*Seeded database with data for categories and posts 

Abi and/or Lora - please review our changes. Thanks!

### CODE REVIEW

File: app/views/categories/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<p>Testing category route</p>
```
Comment (labe) [inline app/views/categories/show.html.erb]: clean up (i.e. remove) plz

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,7 @@
 RailsCraigslist::Application.routes.draw do
+  resources :categories do
```
Comment (labe) [inline config/routes.rb:2]: can be written as simply <code>resources :categories</code>

unless you plan on nesting or adding stuff to the <code>:categories</code> route group

Comment (keccers) [inline config/routes.rb:2]: we do plan on nesting `resources :posts` within categories!

File: app/views/categories/show.html.erb (hunk 1 above)
Comment (keccers) [inline app/views/categories/show.html.erb]: removed testing line and added h2 tag around title.

**Comment by labe** [comment]
hotness :fire:

**Pull request merged**